cosmwasm/Dockerfile.deploy

```diff
@@ -0,0 +1,28 @@
+# This is a multi-stage docker file:
+#  1. The first stage contains the built contracts (wasm files)
+#  2. The second creates a node.js environment to deploy the contracts to devnet
+
+#  local cosmwasm image, built by tilt
+FROM cosmwasm_artifacts AS artifacts
+
+# Contract deployment stage
+FROM node:16-buster-slim@sha256:93c9fc3550f5f7d159f282027228e90e3a7f8bf38544758024f005e82607f546
+
+# Support additional root CAs
+COPY cert.pem* /certs/
+# Node
+ENV NODE_EXTRA_CA_CERTS=/certs/cert.pem
+ENV NODE_OPTIONS=--use-openssl-ca
+# npm
+RUN if [ -e /certs/cert.pem ]; then npm config set cafile /certs/cert.pem; fi
+
+RUN apt update && apt install netcat curl jq -y
+
+WORKDIR /app/tools
+
+COPY --from=artifacts / /app/artifacts
+
+COPY ./tools/package.json ./tools/package-lock.json /app/tools/
+RUN --mount=type=cache,uid=1000,gid=1000,target=/home/node/.npm \
+    npm ci
+COPY ./tools /app/tools
```

Comment:
is the idea here that other cosmos chains in the tilt env will use this dockerfile for deploying contracts? i.e. wormchain

Reply:
Correct, this is used by wormchain and terra2. Could be used by other cosmos chains.